express/src/routes/manage-account.ts

```diff
@@ -18,10 +18,17 @@ router.get('/client-details/:serviceId', async (req, res) => {
     const selfServiceClientId = client.sk.substring("client#".length);
     const serviceId = req.params.serviceId;
     const authClientId = client.clientId;
+
+    let isUpated = false;
```

Comment:
You could just use `req.session.userDetailsUpdated` in place of `isUpdated` on line 30 and do away with the variable and if-block.

However, you can also rely on the falsiness of `undefined` and replace `isUpdated` with `req.session.updatedField` instead of `req.session.userDetailsUpdated`.

And once you've done that, you can just use `updatedField` in the template's if-block instead of `userDetailsUpdated` and reduce the number of things to deal with in the session and variables that need to be passed to the template.